firebase-perf/src/main/java/com/google/firebase/perf/metrics/NetworkRequestMetricBuilder.java

```diff
@@ -293,7 +293,7 @@ public NetworkRequestMetricBuilder setResponseContentType(@Nullable String conte
     if (isValidContentType(contentType)) {
       builder.setResponseContentType(contentType);
     } else {
-      logger.info("The content type of the response is not a valid content-type:" + contentType);
+      logger.debug("The content type of the response is not a valid content-type:" + contentType);
```

Comment:
Should this be warn?

Reply:
My original thought is that this one can be either because `contentType` is extracted from response. We [check `contentType` length and disallow control characters](https://github.com/firebase/firebase-android-sdk/blob/master/firebase-perf/src/main/java/com/google/firebase/perf/metrics/NetworkRequestMetricBuilder.java#L383-L399) and may cause false alarm. 
I updated to `warn` to match that this `contentType` is not expected from Fireperf's standard.